test/nodes/test_tool_http_request.py

```
@@ -0,0 +1,442 @@
+# ===================================================================
+# IGlobal
+# ===================================================================
+
+
+class TestHttpRequestIGlobal:
+    """Test suite for tool_http_request IGlobal."""
+
+    def test_build_guardrails_default_methods(self):
+        """_build_guardrails with no explicit flags should enable default methods."""
+        cfg = {}
+        methods, patterns = IGlobal._build_guardrails(cfg)
```

Ruff (0.15.7)

[warning] Unpacked variable `patterns` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)